test/config.json

```diff
@@ -23,9 +23,37 @@
           "remote": true
         }
       ]
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "firefox",
+          "browser_version": "74.0 beta",
+          "os": "OS X",
+          "os_version": "Catalina",
```

Comment:
Would it be worth adding Firefox on Windows also for the multiple browsers testing while we're at it?

Reply:
Added :)